src/common/snippets/src/shape_inference/shape_inference.cpp

```diff
@@ -52,9 +52,11 @@ const IShapeInferSnippetsFactory::TRegistry IShapeInferSnippetsFactory::registry
         SHAPE_INFER_PREDEFINED(op::HorizonSum, HorizonOpShapeInfer),
         //
         SHAPE_INFER_PREDEFINED(op::LoopBegin, SingleElementShapeInfer),
+        SHAPE_INFER_PREDEFINED(op::PerfCountBegin, SingleElementShapeInfer),
```

Comment:
As we discussed in these two comments: https://github.com/openvinotoolkit/openvino/pull/19493#discussion_r1385183089, https://github.com/openvinotoolkit/openvino/pull/19493#discussion_r1384912667
PerfCountBegin shapeInfer should be the same as for PerfCountEnd. Please fix

Reply:
done, thanks